tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -520,28 +522,30 @@ def get_compute_worker_run_info(self, scheduled_run_id: str):
 
 
 def test_get_compute_worker_runs(mocker: MockerFixture) -> None:
+    dataset_id = generate_id()
+    run_ids = [generate_id(), generate_id()]
     client = ApiWorkflowClient(token="123")
```

Comment:
I just noticed this, doesn't this make an api request to prod if we instantiate `ApiWorkflowClient` without mocking it? Because `ApiWorkflowClient` runs a version check on initialization.

Reply:
You are right. I just updated these test cases.

There are actually some other worse things:
* Every time `from lightly.... import ...` is executed in the test suites, a version check to prod is performed.
* The `MockedApiWorkflowClient` class never really mocked the version API, so these test cases have been hitting prod for a long time.

Reply:
Thanks for investigating! Let's make a follow-up and fix asap.

Reply:
Fix is here #1267